eventdiscovery-sdk/src/main/res/layout/schedjoules_fragment_event_details.xml

```diff
@@ -24,14 +24,78 @@
             <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_marginTop="8dp"
                     android:orientation="vertical">
 
-                <LinearLayout
-                        android:id="@+id/schedjoules_event_detail_vertical_items"
-                        android:orientation="vertical"
+                <include
+                        layout="@layout/schedjoules_view_event_details_venue"
+                        android:id="@+id/schedjoules_event_details_venue_name"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:layout_marginTop="12dp"/>
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginRight="16dp"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginBottom="8dp"/>
+
+                <include
+                        layout="@layout/schedjoules_view_event_details_datetime"
+                        android:id="@+id/schedjoules_event_details_datetime"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginRight="16dp"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginBottom="8dp"/>
+
+                <include
+                        layout="@layout/schedjoules_view_event_details_description"
+                        android:id="@+id/schedjoules_event_details_description"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginRight="16dp"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginBottom="8dp"/>
+
+                <include
+                        layout="@layout/schedjoules_view_event_details_action"
```

Comment:
Yeah, that was intentional at that point, I wanted to get back to styles and the end, also to see how the layout ends up. I've created a style for these event detail item sizes.
I left the 16dp horizontal padding instead of 8dp and then 8dp on the container, to have a greater area for the touch ripple effect.